packages/@react-native-windows/tester/js/examples-win/DisplayNone/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```

Comment:
don't add files under the .vs folder, those are autogenerated by Visual Studio (includes things like local user settings, intellisense cache, etc.)